seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonErrorCode.java

```diff
@@ -19,6 +19,8 @@
 
 /** SeaTunnel connector error code interface */
 public enum CommonErrorCode implements SeaTunnelErrorCode {
+    JSON_OPERATION_FAILED("COMMON-02", "<format> JSON convert/parse '<payload>' operation failed."),
```

Comment:
Would you mind convert other `CommonErrorCodeDeprecated::JSON_OPERATION_FAILED` to new `CommonErrorCode::JSON_OPERATION_FAILED` with another PR like you did in this PR?

Reply:
ACK